opencog/nlp/chatbot/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
-ADD_GUILE_MODULE (chatbot.scm
+ADD_GUILE_MODULE (FILES
+	chatbot.scm
```

Comment:
Is this correct?  The `chatbot.scm` file normally goes one directory lower, than the rest of the files in the module.  In this example,  `chatbot.scm` would normally go into `${DATADIR}/scm/opencog/nlp/`  where guile finds it, and all the other files go into `${DATADIR}/scm/opencog/nlp//chatbot/`  ... or does guile automagically find the "main" entry point somehow?